api/requirements.txt

```diff
@@ -22,7 +22,8 @@ itypes==1.1.0             # via coreapi
 jinja2==2.10              # via coreschema
 jmespath==0.9.3           # via boto3, botocore
 markupsafe==1.0           # via jinja2
-psycopg2==2.7.4
+psycopg2-binary==2.7.4
+psycopg2==2.7.4           # via djangorestframework-hstore
```

Comment:
Rm this line since we have the bin version?

Reply:
I'm not sure why `djangorestframework-hstore` wants it, but this no longer triggers the warning so since it works I'd rather not mess with it.